stripe/src/main/java/com/stripe/android/view/CardInputWidget.java

```diff
@@ -30,43 +29,21 @@
 import com.stripe.android.model.Card;
 import com.stripe.android.StripeTextUtils;
 
-import java.lang.annotation.Retention;
-import java.lang.annotation.RetentionPolicy;
-import java.util.HashMap;
 import java.util.Locale;
-import java.util.Map;
 
+import static com.stripe.android.model.Card.BRAND_RESOURCE_MAP;
 import static com.stripe.android.model.Card.CVC_LENGTH_AMERICAN_EXPRESS;
 import static com.stripe.android.model.Card.CVC_LENGTH_COMMON;
 import static com.stripe.android.model.Card.CardBrand;
+import static com.stripe.android.view.CardInputListener.FocusField.FOCUS_CARD;
+import static com.stripe.android.view.CardInputListener.FocusField.FOCUS_CVC;
+import static com.stripe.android.view.CardInputListener.FocusField.FOCUS_EXPIRY;
 
 /**
  * A card input widget that handles all animation on its own.
  */
 public class CardInputWidget extends LinearLayout {
 
-    @Retention(RetentionPolicy.SOURCE)
```

Comment:
These fields were moved into the `CardInputListener` class, which was extracted from `CardInputWidget` so that both the original and the multiline widget could use it.